### PR TITLE
LibJS: Add String.prototype.anchor & friends

### PR DESCRIPTION
**Pull request by IdanHo**
Adds an implementation of the following StringPrototype methods: anchor, big, blink, bold, fixed, fontcolor, fontsize, italics, link, small, strike, sub, sup.

(This results in 69 new passing test262 test-cases)

### CODE REVIEW

File: Userland/Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 1]
```diff
@@ -722,4 +735,96 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::replace)
...
+{
+    auto& vm = global_object.vm();
+    auto str = string.to_string(global_object);
+    if (vm.exception())
+        return {};
+    StringBuilder builder;
+    builder.append('<');
+    builder.append(tag);
+    if (!attribute.is_empty()) {
+        auto value_string = value.to_string(global_object);
```
Comment (linusg) [inline Userland/Libraries/LibJS/Runtime/StringPrototype.cpp:749]: Missing exception check

File: Userland/Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 2]
```diff
@@ -722,4 +735,96 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::replace)
...
+        value_string.replace("'", "&quot;", true);
```
Comment (linusg) [inline Userland/Libraries/LibJS/Runtime/StringPrototype.cpp:750]: U+0022 is `"`, not `'`

File: Userland/Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 3]
```diff
@@ -722,4 +735,96 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::replace)
...
+        builder.append(' ');
+        builder.append(attribute);
+        builder.append("='");
```
Comment (linusg) [inline Userland/Libraries/LibJS/Runtime/StringPrototype.cpp:753]:
```suggestion
        builder.append("=\"");
```

File: Userland/Libraries/LibJS/Runtime/StringPrototype.cpp [hunk 4]
```diff
@@ -722,4 +735,96 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::replace)
...
+        builder.append(value_string);
+        builder.append('\'');
```
Comment (linusg) [inline Userland/Libraries/LibJS/Runtime/StringPrototype.cpp:755]:
```suggestion
        builder.append('"');
```

**Review by linusg — CHANGES_REQUESTED** [review]

**Comment by IdanHo** [comment]
Oops, fixed :sweat_smile:

**Pull request merged**